box-ui/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
is the use of a env file recommended in a frontend ? Shouldn't we use something like a config.js file that we would import in the components ?